### ISSUE TITLE
Why use softmax with class?

### ISSUE THREAD

**Issue by litingsjj** [experiencor/keras-yolo3#167]
I find yolov2 and yolov3 use same loss function of class.
